projects/plugins/jetpack/modules/notes.php

```diff
@@ -205,6 +202,17 @@ public function admin_bar_menu() {
 		);
 	}
 
+	/**
+	 * Returns the HTML markup for used by notification in top bar
+	 *
+	 * @return string
+	 */
+	private static function get_notes_markup() {
+		return '<span id="wpnt-notes-unread-count" class="wpnt-loading wpn-read"></span>
+<span class="noticon noticon-bell"></span>
+<span class="screen-reader-text">' . __( 'Notifications', 'jetpack' ) . '</span>';
```

Comment:
I would recommend leaving the extra escaping in there as it was:

```suggestion
<span class="screen-reader-text">' . esc_html__( 'Notifications', 'jetpack' ) . '</span>';
```

Reply:
Done in 2452d8a863e2907b93a78677e541e71bf8554266